webhooks-extension/config/extension-deployment.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: webhooks-extension
+  labels:
+    app: webhooks-extension
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: webhooks-extension
+  template:
+    metadata:
+      labels:
+        app: webhooks-extension
+    spec:
+      containers:
+        - name: webhooks-extension
+          image: "DOCKER_REPO/extension:latest"
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 8080
+          livenessProbe:
+            httpGet:
+              path: /liveness
+              port: 8080
+          readinessProbe:
+            httpGet:
+              path: /readiness
+              port: 8080
+          env:
+          - name: PORT
+            value: "8080"
+          - name: INSTALL_NAMESPACE
+            value: INSTALL_NAMESPACE
```

Comment:
rather than having to substitute ... you can actually declare this env var as follows:

```
- name: INSTALL_NAMESPACE
  valueFrom:
    fieldRef:
      fieldPath: metadata.namespace
```

this sets the INSTALL_NAMESPACE from the metadata

Reply:
Cool, I didn't know you could do this! 😄